Bug 1308315 and Bug 1306612 : deleted template and fixed file location

bfallonf opened the pull request:
As per:
https://bugzilla.redhat.com/show_bug.cgi?id=1308315
https://bugzilla.redhat.com/show_bug.cgi?id=1306612

Added the cluster metrics template example for Origin, and ifdef'd it and the Enterprise example. Also, the file location above. 

Also added the bit in Persistent Storage about the type of pv. 

I'll ask for a review in the BZ.

adellape commented on install_config/cluster_metrics.adoc:
```diff
@@ -144,6 +144,8 @@ link:../architecture/additional_concepts/storage.html#persistent-volumes[persist
 volume] and be able to survive a pod being restarted or recreated. This is ideal
 if you require your metric data to be guarded from data loss.
 
+For cluster metrics to work with persistent storage, ensure that the persistent volume has the *RearWriteOnce* access mode. If not, the persistent volume claim will not be able to find the persistent volume, and Cassandra will fail to start.
```
s/RearWriteOnce/ReadWriteOnce/

adellape commented:
@bfallonf One comment inline.

Can you also give the PR title and squashed commit a more descriptive message?

bfallonf commented:
Given the thumbs up in the BZ (BZ 1308315 is a simple fix, so that's good too).

@adellape Should be ok to merge if there's nothing from anybody.

xlr-8 commented:
:+1:

adellape commented:
@bfallonf There's a couple lines that could be wrapped, but LGTM.

bfallonf commented:
*Revision History*

install_config/cluster_metrics.adoc:

Deleted template in install_config/cluster_metrics.adoc#creating-the-deployer-template[deployer template section] and fixed incorrect file location.

mwringe commented:
Just a note that the instruction for metrics is no longer exactly valid any more as it points to the a file location which may not exist on a user's system. We can't assume that the template files will be available at that location on a system running origin since origin can be installed directly via a tar or some other means. This was also brought up in the original bugzilla (https://bugzilla.redhat.com/show_bug.cgi?id=1306612#c12)

A new issue has been opened to address this: https://github.com/openshift/openshift-docs/issues/1656

bfallonf commented:
@mwringe ack. I must have misunderstood.  I'll comment on the issue.
